table.md

```diff
@@ -56,90 +56,90 @@
 | Ingo, stop it already! This is *exactly* the kind of "blame everybody else than yourself" behavior that I was talking about earlier. ... Ingo, look your code in the mirror some day, and ask yourself: why do you think this fixes a "regression"? ...So by trying to make up for vsyscalls only in your numbers, you are basically trying to lie about regressions, and try to cover up the schednuma regression by fixing something else. ... See? That's bogus. When you now compare numbers, YOU ARE LYING. You have introduced a performance regression, and then trying to hide it by making something else go faster. ...The same is true of all your arguments about Mel's numbers wrt THP etc. Your arguments are misleading - either intentionally, of because you yourself didn't think things through. ... | 0.73511300672 |
-| I never *ever* want to see this code ever again. Sorry, but last time was too f*cking painful. | 0.736764336926 |
+| I never *ever* want to see this code ever again. Sorry, but last time was too f\*cking painful. | 0.736764336926 |
 | This looks totally invalid....Your patch is horribly wrong. | 0.740115025892 |
 | No it's not. Please fix your crappy script. First off, that '#' is wrong. It should be a space. | 0.742679263505 |
 | And no, we should *not* play games with "tlb->local.next". That just sounds completely and utterly insane. That's a hack, it's unclear, it's stupid, and it's connected to a totally irrelevant implementation detail, namely that random RCU freeing. Set a flag, for chrissake. | 0.743261689309 |
-| What the F*CK is wrong with people? | 0.74514543967 |
-| ...And a C++ person who says that "(vodi *)0" is just any "void *" is a *fucking*moron*, ...There is absolutely *zero* excuse for the idiotic traditional C++ brain damage of saying "NULL cannot be '(void *)0', because 'void *' will warn". Anybody who says that is lying. ... The C++ people? They are morons, and they never got it, and in fact they spent much of their limited mental effort arguing against it. ... The whole "it's a stronger type system, so NULL has to be 0" is pure and utter garbage. It's wrong. It's stupid. ... Yeah, I'm not a fan of C++. It's a cruddy language. | 0.745451955086 |
+| What the F\*CK is wrong with people? | 0.74514543967 |
+| ...And a C++ person who says that "(vodi *)0" is just any "void *" is a *fucking\*moron*, ...There is absolutely *zero* excuse for the idiotic traditional C++ brain damage of saying "NULL cannot be '(void *)0', because 'void *' will warn". Anybody who says that is lying. ... The C++ people? They are morons, and they never got it, and in fact they spent much of their limited mental effort arguing against it. ... The whole "it's a stronger type system, so NULL has to be 0" is pure and utter garbage. It's wrong. It's stupid. ... Yeah, I'm not a fan of C++. It's a cruddy language. | 0.745451955086 |
```

Comment:
Also here. Sorry for comment spam, just trying to mark all cases for my future reference.

Reply:
Also, looks like the censorship didn't catch this F-:bomb:
:laughing: